application/libraries/account/Authorization.php

```diff
@@ -57,60 +57,38 @@ function is_permitted($permission_keys, $require_all = FALSE)
 
     $this->CI->load->model('account/Acl_permission_model');
 
-    $account_permissions = $this->CI->Acl_permission_model->get_by_account_id($account_id);
+    $account_permissions = array();
+
+    $permissions = $this->CI->Acl_permission_model->get_by_account_id($account_id);
+    foreach ($permissions as $perm) {
+        $account_permissions[] = strtolower($perm->key);
```

Comment:
I've added strtolower to both ``$permission_keys`` and ``$account_permissions`` to keep the same behaviour of ``strtolower($perm->key) == strtolower($key)``